L63-dotnet-grpc-web.md

```diff
@@ -0,0 +1,98 @@
+
+*Startup.cs* configuration file:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddGrpc();
+
+    // Enable gRPC-Web for all services
+    services.AddGrpcWeb(o => o.GrpcWebEnabled = true);
+}
+
+public void Configure(IApplicationBuilder app)
+{
+    app.UseRouting();
+
+    // Add gRPC-Web middleware
+    app.UseGrpcWeb();
+
+    app.UseEndpoints(endpoints =>
+    {
+        endpoints.MapGrpcService<GreeterService>();
+    });
+}
+```
+
+* Add `Grpc.Net.Client.Web`, a new package that acts as a gRPC-Web outgoing proxy for the .NET gRPC client. This package provides a `DelegatingHandler` that translates outgoing gRPC HTTP/2 requests to gRPC-Web. Base64 content is encoded and decoded as content is read. When a gRPC channel is created the developer can choose to configure it to use a HttpClient that has the `GrpcWebHandler`. When the handler is created the developer chooses whether to use `application/grpc-web` or `application/grpc-web-text`.
+
+```csharp
+// Create delegating handler and configure to use application/grpc-web-text
+var grpcWebHandler = new GrpcWebHandler(GrpcWebMode.GrpcWebText, new HttpClientHandler());
+
+var channel = GrpcChannel.ForAddress("https://localhost:5001", new GrpcChannelOptions
+    {
+        HttpClient = new HttpClient(grpcWebHandler)
+    });
+var client = new Greeter.GreeterClient(channel);
+```
+
+### Interoperability
+
+It is critical that any new gRPC-Web client and server implementations are compatible with existing gRPC-Web implementations.
+
+For interop testing, the [gRPC interop test suite](https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md) is executed using grpc-web. The existing grpc-dotnet interop test server and test client have been updated with configuration options to use grpc-web.
+
+Some tests in the test suite can't be run using gRPC-Web (i.e. some auth tests, client and bidirectional streaming tests) and are excluded.
+
+The tests used for gRPC-Web interop testing are:
+
+* empty_unary
+* large_unary
+* server_streaming (grpc-web-text only)
+* custom_metadata
+* special_status_message
+* unimplemented_service
+* unimplemented_method
+* client_compressed_unary
+* server_compressed_unary
+* server_compressed_streaming (grpc-web-text only)
+* status_code_and_message
+
+## Release
+
+gRPC-Web for .NET will be initially released as a public preview. This gives time to ensure the new gRPC-Web implementation is compatible with other implementations, find bugs and technical issues, and get feedback that this solution is meeting developer needs.
```

Comment:
add a note that we've already published grpc-web support as stable in version XYZ?

Reply:
Looks like first stable release was https://www.nuget.org/packages/Grpc.AspNetCore.Web/2.29.0 from May 2020